src/plugins/data_view_field_editor/public/components/field_format_editor/editors/color/color.tsx

```diff
@@ -38,8 +38,8 @@ export class ColorFormatEditor extends DefaultFormatEditor<ColorFormatEditorForm
   constructor(props: FormatEditorProps<ColorFormatEditorFormatParams>) {
     super(props);
     this.onChange({
-      fieldType: props.fieldType,
-    });
+      fieldType: props.fieldType, // FIXME: why add `fieldType` as an EditorFormatParam?
```

Comment:
Could we address this before merging? Or just make a very careful note to follow up? Tons of progress in this PR, it would be nice to nail this down.

Reply:
As it seems, `fieldType` is a valid key of FormatEditorProps, but the object literal should be in the shape of `ColorFormatEditorFormatParams`. 

It may be a long shot, but perhaps this experiment will help shine a light here: https://github.com/elastic/kibana/pull/135879

Reply:
The experimental PR completely removes these lines, and is passing CI. Maybe there isn't automated test coverage of this piece. I will leave the PR open as a reminder to follow up.